test/tests/performance/pbs_sched_perf.py

```diff
@@ -446,20 +446,15 @@ def test_multi_sched_perf(self):
         for sc in self.scheds:
             a = {'scheduling': 'False'}
             self.server.manager(MGR_CMD_SET, SCHED, a, id=sc)
-        for sc in self.scheds:
-            self.scheds[sc].log_match("Leaving Scheduling Cycle",
-                                      starttime=int(start),
-                                      max_attempts=30, interval=1)
         for sc in self.scheds:
             if sc != 'default':
-                filename = self.scheds[sc].attributes['sched_log']
-                sl = PBSSchedulerLog()
-                end = time.time()
-                sl.analyze(filename, start, end, self.server.hostname)
-                cycles = sl.cycles
-                if cycles is None or len(cycles) == 0:
-                    return []
-                dur = cycles[-1:][0].end - cycles[-1:][0].start
+                self.logger.info("searching log for scheduler " + str(sc))
+                log_msg = self.scheds[sc].log_match("Leaving Scheduling Cycle",
+                                                    starttime=int(start),
+                                                    max_attempts=30)
+                endtime = PBSLogUtils.convert_date_time(
+                    log_msg[1].split(';')[0])
+                dur = endtime - start
                 self.perf_test_result(dur, sc + "_cycle_duration", "secs")
                 self.logger.info('Time taken by ' + str(sc) +
                                  ' scheduler to run 1k jobs is ' + str(dur))
```

Comment:
I think this message is too detailed for what we want.  The purpose of this test is to see the difference between running 1 cycle with 5k jobs and 5 cycles with 1k jobs and see the latter is faster.  I don't think we care the individual cycle times for the 5 cycles.  We want to see the difference between the first cycle, and the longest of the 5 cycles.

Reply:
done. @bhroam Please review